src/main/java/Main.java

```diff
@@ -0,0 +1,7 @@
+public class Main {
+
+    public static void main(String[] args) {
+        BankingSystem bankingSystem = new BankingSystem(args[1]);
```

Comment:
What parameter I should use? Can you add it to the readme file?

Reply:
If there are no arguments it will be thrown - `ArrayIndexOutOfBoundsException`. Can you do validation?

Reply:
Why is the `BankingSystem` should aware of the DB name?

Reply:
Updated readme file, made a validation for arguments and removed db name parameter in `BankingSystem` constructor.